src/tests/support.h

```diff
@@ -223,3 +223,4 @@ pgp_key_t *rnp_tests_get_key_by_id(rnp_key_store_t *  keyring,
                                    pgp_key_t *        after);
 pgp_key_t *rnp_tests_get_key_by_fpr(rnp_key_store_t *keyring, const std::string &keyid);
 pgp_key_t *rnp_tests_key_search(rnp_key_store_t *keyring, const std::string &keyid);
+size_t     get_longest_line_length(const std::string &str);
```

Comment:
This function seems to be used only in single place, so can be moved there and made static.